radar/umrr_can_publisher:
  # parameters to send to sensor
  # antenna_mode: {section: 2010, parameter: 0, value: 2}
  # center_frequency: {section: 3016, parameter: 5, value: 2, dim: 0}
  # necessary parameters
  can_spec: "$(find umrr_driver)/can_spec/Automotive_Can_Spec_TargetList_V5.json"
  can_socket: "can0"
  legacy_mode: False
  # omit empty strings to be set by node (must not be edited by user)
  sensor_type: ""
  software_vers: ""
  curr_antenna: ""





